extensions/sql-database-projects/src/models/project.ts

```diff
@@ -155,6 +160,15 @@ export class Project implements ISqlProject {
 			this.projFileXmlDoc!.documentElement.getElementsByTagName(constants.PropertyGroup)[0]?.appendChild(newProjectGuidNode);
 			await this.serializeToProjFile(this.projFileXmlDoc);
 		}
+
+		// get output path
+		try {
+			const outputPath = this.projFileXmlDoc!.documentElement.getElementsByTagName(constants.OutputPath)[0].childNodes[0].nodeValue!;
+			this._outputPath = path.join(utils.getPlatformSafeFileEntryPath(this.projectFolderPath), utils.getPlatformSafeFileEntryPath(outputPath));
+		} catch (e) {
+			// If output path isn't specified in .sqlproj, set it to the default output path .\bin\Debug\
```

Comment:
can we check the path fist and set to default instead of catching an exception?

Reply:
updated to be an if else checking if any OutputPath nodes were found